.github/workflows/ci.yml

```diff
@@ -19,7 +25,8 @@ jobs:
           - python-version: '3.9'
           - python-version: '3.10'
           - python-version: '3.11'
-          - python-version: '3.11'
+          - python-version: '3.12'
+          - python-version: '3.12'
```

Comment:
do we need to do python3.12 twice?

Reply:
first line is just Python `3.12`

the second line is `3.12` + `install-extra: '[sentry]'`

(this was the same as in the original `3.11` and `3.11` + `install-extra: '[sentry]'`)

Reply:
ah right